### PR TITLE
Add get characters response

### PR DESCRIPTION
**Pull request by andrew-brainerd**
Added a new function to get the associated characters for a given game. As I'm typing this, I'm remembering that @ajbowen249 wanted to keep characters in a separate collection, so the line where we declare the `characters` to pass back will need to change to another database call.

### CODE REVIEW

**Comment by ajbowen249** [comment]
@andrew-brainerd, I'm not married to keeping characters separate. it may end up causing more trouble than it is worth in the long run.

File: src/data/games.js [hunk 1]
```diff
@@ -43,9 +50,71 @@ const addLog = async (gameId, message) => {
...
+    id: uuidv1(),
+    timestamp: new Date(),
+    ...message
+  }
+});
+
+const parseUserInput = async userInput => {
+  const { messages: { gameId, message } } = userInput || {};
+
+  if (message === 'login' || message === 'signin') {
```
Comment (ajbowen249) [inline src/data/games.js]: Not for this PR, but I added a [card](https://app.gitkraken.com/glo/board/Xmamiq5zLQARKWCH/card/XnKUBTHfCwASQ0Ho) for making this a little more usable.

File: src/routes/api/games.js [hunk 2]
```diff
@@ -35,8 +27,8 @@ games.post('/', validator.body(postGameBody), async (req, res) => {
 });
 
 const getGameQuery = Joi.object({
-  pageNum: Joi.number().required(),
-  pageSize: Joi.number().required(),
+  pageNum: Joi.number(),
```
Comment (ajbowen249) [inline src/routes/api/games.js]: Oops...sorry :sweat_smile:

File: src/data/games.js (hunk 1 above)
Comment (andrew-brainerd) [reply src/data/games.js]: Oh absolutely. This is some pathetic shit so far. I've been skirting the issue, so we can have a larger group discussion about how our "commands" are going to be formatted and parsed.

File: src/routes/api/games.js (hunk 2 above)
Comment (andrew-brainerd) [reply src/routes/api/games.js]: Not on you at all. I just realized they have default values, so they aren't actually required.

**Comment by andrew-brainerd** [comment]
@ajbowen249 As far as the character separation, I can see it both ways. Like, if we nest them under the game, we end up getting all the character data when we pull the game data down, and vice versa. That may not truly be a big deal. Either way, Mongo being so flexible means we can probably put them in or pull them out later if we run into extenuating circumstances.

**Pull request merged**